docs/slides/day2.md

```diff
@@ -0,0 +1,577 @@
+  prints `"it's true"` if `x` is greater than `0` and `y` is less than `0`.
+
+* **Challenge** - come up with a different way to write the conditional
+  statement that behaves the same. Hint: use negation (`!`).
+
+```ruby
+x = 2
+y = -2
+z = 4
+```
+
+--
+
+```ruby
+# Option 1
+if x > 0 && y < 0
+    puts "it's true!"
+end
+```
+
+--
+
+```ruby
+# Option 2
+if !(x <= 0 || y >= 0)
+    puts "it's true!"
+end
+```
+
+[De Morgan's laws](https://github.com/Ada-Developers-Academy/jump-start-live/blob/master/lessons/day2/boolean_expressions.md#demorgans-laws)
+
+---
+
+# What does the code below do?
+```ruby
+print "What size drink would you like? (SHORT or TALL or GRANDE): "
+order = gets.chomp.upcase
+
+if order == "SHORT"
+    puts "8"
+elsif order == "TALL"
+    puts "12"
+else order == "GRANDE"
+    puts "16"
+end
+```
+
+???
+
+* Everything besides short and tall fall in the `else` block
+* `order == "GRANDE"` is actually a statement within the else, but it was
+  mistakenly put on the wrong line
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+What size drink would you like? (SHORT or TALL or GRANDE): short
+8
+```
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+What size drink would you like? (SHORT or TALL or GRANDE): grande
+16
+```
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+What size drink would you like? (SHORT or TALL or GRANDE): no thank you
+16
+```
+
+---
+
+# What does the code below do?
+```ruby
+print "What size drink would you like? (SHORT or TALL or GRANDE): "
+order = gets.chomp.upcase
+
+if order == "SHORT"
+    puts "8"
+elsif order == "TALL"
+    puts "12"
+*else order == "GRANDE"
+    puts "16"
+end
+```
+
+---
+
+# What does the code below do?
+```ruby
+print "What size drink would you like? (SHORT or TALL or GRANDE): "
+order = gets.chomp.upcase
+
+if order == "SHORT"
+    puts "8"
+elsif order == "TALL"
+    puts "12"
+else
+*   order == "GRANDE"
+    puts "16"
+end
+```
+
+--
+
+But this was probably meant to be an `elsif` statement
+
+---
+
+# What does the code below do?
+```ruby
+print "What size drink would you like? (SHORT or TALL or GRANDE): "
+order = gets.chomp.upcase
+
+if order == "SHORT"
+    puts "8"
+elsif order == "TALL"
+    puts "12"
+elsif order = "GRANDE"
+    puts "16"
+else
+    puts "unknown size"
+end
+```
+
+--
+
+```console
+brooks@ada:~$ ruby exercise.rb
+exercise.rb:8: warning: found `= literal' in conditional, should be ==
+What size drink would you like? (SHORT or TALL or GRANDE): grande
+16
+```
+
+```console
+brooks@ada:~$ ruby exercise.rb
+exercise.rb:8: warning: found `= literal' in conditional, should be ==
+What size drink would you like? (SHORT or TALL or GRANDE): no thank you
+16
+```
+
+---
+
+# What does the code below do?
+```ruby
+print "What size drink would you like? (SHORT or TALL or GRANDE): "
+order = gets.chomp.upcase
+
+if order == "SHORT"
+    puts "8"
+elsif order == "TALL"
+    puts "12"
+*elsif order = "GRANDE"
```

Comment:
Is this * a typo? Maybe I'm just missing how you're using the * to highlight

Reply:
@beccaelenzil Yes that's what's happening, I showed the functionality in https://github.com/Ada-Developers-Academy/jump-start-live/pull/31#discussion_r362360895, but I suppose I didn't actually explain how, functionally, it works.

You can highlight lines by making the first character in the line an asterisk